### PR TITLE
[FEATURE] Switch Transfer Provider to Redux

### PR DESCRIPTION
**Pull request by ericlowturing**
[Associated Notion Card](https://www.notion.so/9f3775679657499487a1a06dfa58d381?v=82d95bfda1c84891b4c063f7415e414d&p=def5561992ea4f37866c9206cb8d4745)

-----

## 1. Pull request checklist

- [ ] Notion card has been moved to the Code Review column
- [ ] Notion card has a link to this PR
- [ ] A reviewer has been assigned to the Notion card

### CODE REVIEW

File: src/store/store.mockData.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { notificationsReady } from './notifications/notifications.mockData';
+import { REDUCER_NAME as CURRENCY_REDUCER_NAME } from './currency/reducer';
+import { currencyReady } from './currency/currency.mockData';
+import { REDUCER_NAME as MINT_REDUCER_NAME } from './mint/reducer';
+import { mintReady } from './mint/mint.mockData';
+import { REDUCER_NAME as TRANSFER_REDUCER_NAME } from './transfer/reducer';
+import { transferReady } from './transfer/transfer.mockData';
+
+export const storeReady = {
+	[NOTIFICCATION_REDUCER_NAME]: notificationsReady,
```
Comment (colbr) [inline src/store/store.mockData.ts:12]: Spelling error, should be `NOTIFICATION_REDUCER_NAME` (this one is made a couple times so just `CTRL + F` them all)

**Review by colbr — CHANGES_REQUESTED** [review]
Looking good apart from one small spelling nit.

I'm not hugely familiar with Redux, @padala would you be able to give this a skim too?

File: src/store/store.mockData.ts (hunk 1 above)
Comment (ericlowturing) [reply src/store/store.mockData.ts]: Fixed already. I really appreciate your quick reviews

**Review by colbr — APPROVED** [review]
Seems to be working well, get @padala to have a skim over this one too before merging

**Comment by ericlowturing** [comment]
> 

I will merge it now to avoid merge conflicts with my current redux projects. Please review at any time, but it is urgent for me to get merged. And I hope you to review and leave feedback asap. Thanks

**Pull request merged**